tests/DfE.FindInformationAcademiesTrusts.Data.AcademiesDb.Faker/Fakers/GiasGroupLinkFaker.cs

```diff
@@ -0,0 +1,30 @@
+using System.Globalization;
+using Bogus;
+using DfE.FindInformationAcademiesTrusts.Data.AcademiesDb.Models.Gias;
+
+namespace DfE.FindInformationAcademiesTrusts.Data.AcademiesDb.Faker.Fakers;
+
+public class GiasGroupLinkFaker
+{
+    private readonly Faker<GiasGroupLink> _groupLinkFaker;
+    private readonly DateTime _refDate;
+
+    public GiasGroupLinkFaker(DateTime refDate)
+    {
+        _groupLinkFaker = new Faker<GiasGroupLink>("en_GB");
+        _refDate = refDate;
+    }
+
+    public GiasGroupLink Generate(string uid, string joinedDate, string urn)
+    {
+        var fakeGroupLink = _groupLinkFaker
+            .RuleFor(t => t.JoinedDate, f => f.Date.Between(
```

Comment:
I don't think we need a rule for this (don't really want to modify the `Faker` object on every call to `Generate()`. Can we instead directly set the property on the generated object?

Reply:
shall I just remove the faker entirely then and create a new group?

Reply:
going to look at adding a similar pattern as in `SetLocalAuthorities` and `SetUid` on the `GiasEstablishmentFaker`